packages/schema/src/config/common.ts

```diff
@@ -424,12 +429,13 @@ export default defineUntypedSchema({
    */
   alias: {
     $resolve: async (val: Record<string, string>, get): Promise<Record<string, string>> => {
-      const [srcDir, rootDir, assetsDir, publicDir, buildDir] = await Promise.all([get('srcDir'), get('rootDir'), get('dir.assets'), get('dir.public'), get('buildDir')]) as [string, string, string, string, string]
+      const [srcDir, rootDir, assetsDir, publicDir, buildDir, sharedDir] = await Promise.all([get('srcDir'), get('rootDir'), get('dir.assets'), get('dir.public'), get('buildDir'), get('dir.shared')]) as [string, string, string, string, string, string]
       return {
         '~': srcDir,
         '@': srcDir,
         '~~': rootDir,
         '@@': rootDir,
+        '#shared': resolve(rootDir, sharedDir),
```

Comment:
sorry I know its been merged, but I was just reviewing the changes.

I was wondering, do these alias resolution keys need to be static, especially in this case, where, say, I configure the shared dir to be `common`, so my imports would still be `#shared` (that's fine, I suppose I don't make the rules! but maybe its a little friction in that dev experience), or would we rather make it align to the dir name:

```suggestion
        [`#${sanitisedSharedDirName}`]: resolve(rootDir, sharedDir),
```

Reply:
I'd say thst it is better to keep them static for consistency. You can always define a custom alias if you want.

Reply:
fair enough!